src/js/parsers/captions/srt.js

```diff
@@ -50,7 +50,7 @@ define([
                 entry.begin = _seconds(line.substr(0, index));
                 entry.end   = _seconds(line.substr(index + 5));
                 // Remaining lines contain the text
-                entry.text = array.slice(idx + 1).join('<br/>');
+                entry.text = array.slice(idx + 1).join('\r\n');
```

Comment:
@hghazzi Not for line breaks. Some other [HTML tags](https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#italics-and-underline) are honored, though.